s3/src/main/scala/tamer/s3/S3Configuration.scala

```diff
@@ -11,14 +11,15 @@ import java.time.{Duration, Instant}
 import scala.util.hashing.MurmurHash3.stringHash
 
 final case class S3Configuration[
+    R,
     K <: Product: Codec: SchemaFor,
     V <: Product: Codec: SchemaFor,
     S <: Product: Codec: SchemaFor
 ](
     bucketName: String,
     prefix: String,
     tamerStateKafkaRecordKey: Int,
-    transducer: ZTransducer[Any, TamerError, Byte, V],
```

Comment:
This is it. At some point this was reverted to `Any` but it was really old code, not sure when this happened.